proto/lava/common/fixationEntry.proto

```diff
@@ -14,6 +14,18 @@ message Entry {
     bool is_latest = 7; // is this entry the latest entry now?
 }
 
+message Entries {
```

Comment:
I prefer to make it clear this used purely for genesis by renaming to e.g. "GenesisEntryIndex" - that way we know it's the equivalent of an EntryIndex that is used in Genesis.